editoast/src/views/stdcm_logs.rs

```
@@ -320,8 +320,16 @@ mod tests {
     }
 
     async fn execute_stdcm_request(app: &TestApp, user: Option<&subject::User>) -> String {
-        let small_infra = create_small_infra(&mut app.db_pool().get_ok()).await;
         let timetable = create_timetable(&mut app.db_pool().get_ok()).await;
+        // create an infra and add the provided user as infra owner
+        let small_infra = create_small_infra(&mut app.db_pool().get_ok()).await;
+        if let Some(user) = user {
+            let _ = app
+                .regulator()
+                .grant_infra_owner_unchecked(user.id, small_infra.id)
+                .await;
```

use expect to avoid silent failures